gubernator/github/classifier.py

```diff
@@ -205,13 +171,41 @@ def classify(events, statuses=None):
                 'comments': [{'user': str name, 'comment': comment, 'timestamp': str iso8601}],
                 'xrefs': list of builds referenced (by GCS path),
             }
-    '''
+    """
+    if events_iterator is not None:
+        events = next(events_iterator)
```

Comment:
Do we still need events_iterator?

Also there's still no protection from `next(events_iterator)` raising a `StopIteration`